samples/drivers/watchdog/boards/lp_em_cc2340r5.overlay

```diff
@@ -0,0 +1,3 @@
+&wdt0 {
+	status = "okay";
```

Comment:
Can't this be moved to the board dts file?

Reply:
It could be moved, but for some applications and user scenarios might not be needed

Reply:
If an application doesn't need the watchdog, it should just not enable it via Kconfig. We typically set all supported peripherals in the board dts file to "okay", unless there are some conflicts.

Reply:
Enable by default for the board now